### ISSUE TITLE
MCQ Assessment #3

### ISSUE THREAD

**Issue by abby-albert** [abby-albert/s2#6]
# MCQ Corrections and Takeaways

## Score
62/70 

## Question # 3
<img width="948" alt="image" src="https://github.com/abby-albert/s2/assets/142523000/9e553217-35e0-45fe-a074-e1e16a537938">
B is incorrect. The variable second, while initially assigned the value 200, is later assigned the value of temp, which is 100. The correct answer was A. The variable first is initially assigned the value 100 and the variable second is initially assigned the value 200. Next, temp is assigned the value of first, which is 100. Next, second is assigned the value of temp, which is 100. Last, first is assigned the value of second, which is 100. After execution, both first and second have the value 100.

## Question #13
<img width="958" alt="image" src="https://github.com/abby-albert/s2/assets/142523000/62cc5f58-767b-411a-90ce-d4c44a37b3a1">
Answer B was incorrect. Having multiple team members review components of the innovation throughout the process can help in the identifying and correcting errors. C was correct. While effective collaboration often benefits from a diversity of talents and perspectives, collaborators still need to develop strategies to resolve differences of opinion.

## Question #23
<img width="951" alt="image" src="https://github.com/abby-albert/s2/assets/142523000/39146187-1074-4792-805d-26385db1bbee">
A was incorrect. The Internet uses standardized, open (nonproprietary) communication protocols. B was the correct answer. The Internet was designed to be scalable, using open protocols to easily connect additional computing devices to the network.

## Question #47
<img width="973" alt="image" src="https://github.com/abby-albert/s2/assets/142523000/64e9cc79-c52e-4339-a76c-766ff134ed22">
Answer A was incorrect. This expression will always evaluate to false because prcRange cannot be equal to both "lo" and "med". Answer B was correct. This expression evaluates to true only for restaurants with the correct price range (when prcRange equals "lo" or "med") and the correct customer rating (when avgRating ≥ 4.0).

## Question #48 
<img width="937" alt="image" src="https://github.com/abby-albert/s2/assets/142523000/cf45473e-7516-497a-8555-cdb1061e7ad4">
Answer C was incorrect. Tricking a user into providing personal information is an example of a phishing attack. While this type of attack can be used to obtain personal information, it does not allow unauthorized individuals to intercept information transmitted on a network. Answer A was correct. A rogue access point is a wireless access point that gives unauthorized access to secure networks. Data sent over public networks can be intercepted, analyzed, and modified. One way that this can happen is through a rogue access point.

## Question #55
<img width="938" alt="image" src="https://github.com/abby-albert/s2/assets/142523000/145e4292-06f0-4dab-80ef-05d65f826bab">
Answer A was incorrect. This code segment assigns the value of the last element of the list to the variable temp, then removes the last element of the list, then appends temp to the end of the list. The resulting list is the same as the original list. Answer C was correct. This code segment assigns the value of the last element of the list to the variable temp, then removes the last element of the list, then inserts temp as the first element of the list. 

## Question #66
<img width="992" alt="image" src="https://github.com/abby-albert/s2/assets/142523000/a8780352-56f9-4bfe-9223-0f32cf5a349b">
C is incorrect. This code segment does not work as intended. For example, if timer is greater than 60, bonus is assigned 1500 in the first IF block. Then bonus is assigned 1000 in the second IF block. As a result, bonus will be assigned 1000 instead of the intended 1500. The correct answer was D. In this code segment, if timer is greater than 60, bonus is assigned 1500 in the first IF block. If timer is between 30 and 60, inclusive, bonus is assigned 1000 in the second IF block. If timer is less than 30, bonus is assigned 500 in the third IF block. The correct number of bonus points is assigned to bonus for all possible values of timer.

## Question #67
<img width="957" alt="image" src="https://github.com/abby-albert/s2/assets/142523000/f234ea75-3ecd-42db-90b6-3e50fa12a922">
Answer C was not one of the answers Answers. This code segment simulates a spinner in which there is a 1/4  chance of "blue", a 1/4 chance of "orange", and a 1/4 chance of "purple". However, the given spinner has a 1/4 chance of "blue", a 3/8 chance of "orange", and a 3/8 chance of "purple". Answers A and D were correct. Correct. For this spinner, there is a 1/4 chance of "blue". The remaining 3/4 of the time, "orange" and "purple" are equally likely. If the first call to RANDOM returns 1 (which occurs 1/4 of the time), the code segment prints "blue". Otherwise, if the second call to random returns 1 (which occurs 1/2 of the time that "blue" does not occur), the code segment prints "orange". The other 1/2 of the time that "blue" does not occur, the code segment prints "purple". For this spinner, there is a 1/4 chance of "blue". The remaining 3/4 of the time, "orange" and "purple" are equally likely. The variable spin is set to a random value between 1 and 4, inclusive. If spin is 1 (which occurs 1/4 of the time), the code segment prints "blue". Otherwise, spin is set to a random value between 1 and 2, inclusive. If spin is 2 (which occurs 1/2 of the time that "blue" does not occur), the code segment prints "orange". The other 1/2 of the time that "blue" does not occur, the code segment prints "purple".

## Takeaways 
These are the topics that I missed and need to review to better my overall understanding for the AP exam
- 1.1
- 2.4
- 3.1 
- 3.5
- 3.6
- 3.7
- 4.1
- 5.6

## Conclusion
In conclusion, I’ve identified areas where I need to improve for the AP exam. By reviewing those topics, I aim to strengthen my understanding and perform better on future assessments. I have also identified that areas that I am strong in, and I am satisfied with my score.
